src/org/traccar/notification/FormTypeEventForwarder.java

```diff
@@ -0,0 +1,24 @@
+package org.traccar.notification;
+
+import org.traccar.model.Event;
+import org.traccar.model.Position;
+
+import com.ning.http.client.AsyncHttpClient.BoundRequestBuilder;
+
+public class FormTypeEventForwarder extends EventForwarder {
+
+    @Override
+    protected String getContentType() {
+        return "application/x-www-form-urlencoded; charset=UTF-8";
+    }
+
+    @Override
+    protected void setContent(Event event, Position position, BoundRequestBuilder requestBuilder) {
+        if (!getAdditionalParams().equals("")) {

```

Comment:
Use `isEmpty` instead.

Reply:
I just followed the existing convention

header = Context.getConfig().getString("event.forward.header", "");
...
 if (!header.equals("")) {

Reply:
You followed bad example. That place should be fixed as well.